pkg/model/stack.go

```diff
@@ -675,10 +675,12 @@ func (stack *Stack) mergeServices(otherStack *Stack) *Stack {
 	return stack
 }
 
-func (stack *Stack) GetServices() map[string]bool {
+func (stack *Stack) GetServicesWithBuildSection() map[string]bool {
 	result := make(map[string]bool)
-	for service := range stack.Services {
-		result[service] = true
+	for name, service := range stack.Services {
+		if service != nil && service.Build != nil {
```

Comment:
@jLopezbarb This is not a robust way of checking which services have a build section, because it could be an empty struct. Do you know another way to check if its not _empty_